docs/config.md

```diff
@@ -868,6 +870,44 @@ The key through which access to the pinning service is granted
 
 Type: `string`
 
+#### `Pinning.RemoteServices: Policies`
+
+Contains additional opt-in policies for the remote pinning service
+
+##### `Pinning.RemoteServices: Policies.MFS`
+
+When this policy is enabled it will be following changes to MFS
+and updating the pin for MFS root on the configured remote service.
+
+Pin request to the remote service is sent only when MFS root changed
+and time passed since the previous pin is bigger than `RepinInterval`.
```

Comment:
```suggestion
and time has passed, since the previous pin is bigger than `RepinInterval`.
```
This is unclear to me. Does "since" mean "because" in this case?